partner_contact_lang/tests/test_partner_contact_lang.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Tecnativa - Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 Tecnativa - Vicent Cubells <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+
+
+class TestPartnerContactLang(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestPartnerContactLang, cls).setUpClass()
+        cls.ResPartner = cls.env['res.partner']
+        cls.partner = cls.ResPartner.create({
+            'name': 'Partner test',
+            'lang': 'en_US',
+        })
+        cls.contact = cls.ResPartner.create({
+            'name': 'Contact test',
+            'lang': False,
+            'parent_id': cls.partner.id,
+        })
+
+    def test_onchange_parent_id(self):
+        self.contact.parent_id = False
+        res = self.contact.onchange_parent_id()
+        self.assertIsNone(res)
+        self.contact.parent_id = self.partner
+        res = self.contact.onchange_parent_id()
+        self.assertEqual(res.get('value', {}).get('lang'), 'en_US')
+
+    def test_write_parent_lang(self):
+        self.partner.lang = False
```

Comment:
@cristinamartinrod Why `self.partner.lang` false here? 
And Added [here]( https://github.com/OCA/partner-contact/pull/636/files#diff-81fadbe02bdd60101996d81bd5742cbaR33), In created time You have added already [here](https://github.com/OCA/partner-contact/pull/636/files#diff-81fadbe02bdd60101996d81bd5742cbaR15)

Reply:
It's for using same setup record for testing other things

Reply:
Yes, But here already added in creating time, I  Created Partner then already add lang.

Reply:
This is for being sure this has been emptied before, but OK to remove it.